packages/identity-app/src/Identity.tsx

```diff
@@ -57,13 +57,13 @@ export class Identity extends React.PureComponent<Props, State> {
                   <WithSpace>
                     <Input
                       label='Seed'
-                      onChange={this.handleInputSeed}
+                      onChange={(e) => this.handleInputSeed}
```

Comment:
Quick fly-by comment - I'm not 100% convinced that this is the right approach, basically we create the closure each time we render, generally not the best approach. If you have issues with the `this` binding, this works inside the class -

```
handler = (event) => {
... do stuff
}
```

as opposed to

```
handler (event) {
... do stuff
}
```